concept_map.puml

```diff
@@ -53,12 +60,20 @@ CI -[hidden]down-> Hosting
 Hosting -[hidden]right-> Authentication
 Authentication -[hidden]right-> Firestore
 
-Firebase -[hidden]right-> Core
+Web --up--> Firestore : reads metrics and \l stores app config
+Web --up--> Authentication : authenticates
+Web --up--> Hosting : deployed on
+Web --up--> Analytics : reports analytics data
+Web -down-> Sentry : reports about errors
+
+DartCloudFunctions -up-> CloudFunctions : deployed on
+DartCloudFunctions -[hidden]-> Core : uses
+DartCloudFunctions -up-> Firestore : creates aggregation \l documents
 
-Web -up-> Firestore : reads metrics and \l stores app config
-Web -up-> Authentication : authenticates
-Web -up-> Hosting : deployed on
+CLI --> Web : deploys
```

Comment:
Not sure the CLI deploys the Web only. Instead, what if we rename CLI into `Deploy CLI` on the diagram? Then, keep the `CLI --> Core : uses` but remove the `deploys` line:
```suggestion
```
What do you think?